tensorflow/python/data/experimental/kernel_tests/compression_ops_test.py

```diff
@@ -76,6 +102,30 @@ def testDatasetCompression(self, element):
     dataset = dataset.map(lambda x: compression_ops.uncompress(x, element_spec))
     self.assertDatasetProduces(dataset, [element])
 
+  @combinations.generate(combinations.times(
```

Comment:
Can you add the eager_only objects to the combinations of the existing tests? You can add the combinations like

```
combinations.generate(
  combinations.times(
    test_base.default_test_combinations(), combinations.combine(element=_test_objects()) +
  combinations.times(
    test_base.eager_only_combinations(), combinations.combine(element=_eager_only_test_objects()))
```